packages/forms/src/Components/Repeater.php

```diff
@@ -440,6 +449,15 @@ public function getCachedExistingRecords(): Collection
         );
     }
 
+    public function getItemLabel(string $uuid): ?string
+    {
+        $itemData = data_get($this->getLivewire(), "{$this->getStatePath()}.{$uuid}");
+
+        return $this->evaluate($this->itemLabel, [
+            'itemData' => $itemData,
```

Comment:
I think we should potentially just override `state` instead of using an additional `itemData` variable

Reply:
@danharrin Ahh yeah much better! So just change the `itemData` to `state` would do the trick?

Reply:
Might there be any edge case where people would want to have the whole repeaters state? 😅

Reply:
I think that is probably unexpected behaviour though to return the entire repeater, state is most useful as the current item here

If they really need all items they can just use `$component->getState()`

Reply:
Alright. `$state` is good ;)